backstage/packages/backend/src/plugins/scaffolder/actions/provisioner.ts

```diff
@@ -83,6 +75,40 @@ export const parseEmailInput = (str?: string): string[] => {
   return Array.from(set);
 };
 
+/**
+ * Returns the User's Google Cloud email address.
+ */
+const getGoogleCloudEmailsByRefs = async (
+  catalogApi: CatalogApi,
+  entityRefs: string[],
+  token: string,
+): Promise<string[]> => {
+  const { items } = await catalogApi.getEntitiesByRefs(
+    { entityRefs, fields: ['spec.profile.email'] },
+    { token },
+  );
+
+  const result = [];
```

Comment:
Can we change this to a set? I was able to insert duplicate emails. Sorry if the image is not clear. Tried to redact the emails.
![image](https://github.com/PHACDataHub/sci-portal/assets/48633735/a7963907-6438-4cfa-a2bb-edbd697fd420)

Reply:
Can you add the same person more than once from the UI? If yes, then we'd probably want to see if there's a `ui:option` to make the inputs unique.

Also, how'd you end up without an email address and hit this line?
https://github.com/PHACDataHub/sci-portal/pull/377/files#diff-3fe11c9463a345ad0d1f05460731c7c94732527d0d007203c0e302283b505b69R296

We can also move the `uniq` out so we conditionally prepend, and always `uniq`.